common/supplemental/supplementalData.xml

```diff
@@ -1712,6 +1717,7 @@ XXX Code for transations where no currency is involved
 		<language type="ilo" territories="PH" alt="secondary"/>
 		<language type="inh" scripts="Cyrl"/>
 		<language type="inh" scripts="Arab Latn" territories="RU" alt="secondary"/>
+		<language type="io" scripts="Latn" alt="secondary"/>
```

Comment:
secondary with no primary, see comment below.

Reply:
Same comment for other new secondaries.

Reply:
This is auto-generated. TBH I don't understand why the ConvertLanguageData script did this for this entry but not others -- I suspect because its missing territory data. I'll have to edit ConvertLanguageData.

Reply:
I updated the script so it will no longer demote historical scripts/historical languages.